docs/pages/algorithms/clustering.mdx

```diff
@@ -0,0 +1,586 @@
+```
+
+# Large Fence
+
+## Info
+
+- Points: 169,038
+- Area: 754,594 km²
+- Both of the legacy algorithms have been excluded from this example due to the amount of time it would take to run them.
+
+## Results
+
+```
+# Fastest
+      [STATS] =================================================================
+      || [AREA] Munich | Fastest | Radius                                    ||
+      || [POINTS] Total: 169038 | Covered: 167788                            ||
+      || [CLUSTERS] Total: 21931 | Avg Points: 7                             ||
+      || [BEST_CLUSTER] Amount: 3 | Point Count: 39                          ||
+      || [TIMES] Clustering: 0.20s | Routing: 0.00s | Stats: 0.95s           ||
+      || [MYGOD_SCORE] 67043                                                 ||
+      =========================================================================
+# Fast
+      [STATS] =================================================================
+      || [AREA] Munich | Fast | Radius                                       ||
+      || [POINTS] Total: 169038 | Covered: 162747                            ||
+      || [CLUSTERS] Total: 13554 | Avg Points: 12                            ||
+      || [BEST_CLUSTER] Amount: 1 | Point Count: 48                          ||
+      || [TIMES] Clustering: 6.48s | Routing: 0.00s | Stats: 0.92s           ||
+      || [MYGOD_SCORE] 46953                                                 ||
+      =========================================================================
+# Balanced
+      [STATS] =================================================================
+      || [AREA] Munich | Balanced | Radius                                   ||
+      || [POINTS] Total: 169038 | Covered: 164028                            ||
+      || [CLUSTERS] Total: 13358 | Avg Points: 12                            ||
+      || [BEST_CLUSTER] Amount: 1 | Point Count: 47                          ||
+      || [TIMES] Clustering: 21.16s | Routing: 0.00s | Stats: 0.94s          ||
+      || [MYGOD_SCORE] 45084                                                 ||
+      =========================================================================
+# Better
+      [STATS] =================================================================
+      || [AREA] Munich | Better | Radius                                     ||
+      || [POINTS] Total: 169038 | Covered: 164297                            ||
+      || [CLUSTERS] Total: 13048 | Avg Points: 12                            ||
+      || [BEST_CLUSTER] Amount: 1 | Point Count: 49                          ||
+      || [TIMES] Clustering: 125.90s | Routing: 0.00s | Stats: 0.94s         ||
+      || [MYGOD_SCORE] 43885                                                 ||
+      =========================================================================
+```
+
+# Conclusion
+
+|-------|-----------|------|--------|
+| Fence | Algorithm | Time | Score |
```

Comment:
This isn't formatting properly in Nextra's markdown rendering. Might need to add new lines between the tables? 

Also recommend adding the Time unit (seconds)